### PR TITLE
Match the Show changelog button with the design

### PR DESCRIPTION
**Pull request by anthonydillon**

### CODE REVIEW

**Comment by anthonydillon** [comment]
## QA
- Add an application
- Enter deployment flow
- Scroll down to the "Applications to be deployed" section
- Check the Show changelog button matches the design [1]

[1] https://raw.githubusercontent.com/CanonicalLtd/juju-design/master/Deployment%20flow/Standard/5.png?token=ABWRnkUJu-hMxSNsVU1ypPuQd_pYpQVfks5YR9fywA%3D%3D
_Apart from the white background as that is not how the button component was build. We don't want to much bespoke styling overriding component styles. Therefore that will need to be discussed and may result in an update to our button component_

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6280/
Test PASSed.

File: jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js [hunk 1]
```diff
@@ -304,11 +304,13 @@ YUI.add('deployment-flow', function() {
       return (
         <span className="deployment-flow__service-title">
           Applications to be deployed
-          <juju.components.GenericButton
-            action={this._toggleChangelogs}
-            type="base"
-            title={this.state.showChangelogs ?
-              'Hide changelog' : 'Show changelog'} />
+          <div className="right">
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: Was this extra div necessary? You can pass in the extra `right` class in with the `type` field below.

Comment (anthonydillon) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: The type adds logic. I thought the manipulation might sting someone later so better to keep the "type" stuff consistent. Happy to change if that is not a prob?

Comment (hatched) [inline jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js]: Oh, I'd prefer in this branch.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6292/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6293/
Test PASSed.

**Comment by anthonydillon** [comment]
@hatched done. Ready for another review.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/6299/
Test PASSed.

**Pull request closed without merging**